aws_generated_data/commands/rds_eol.py

```diff
@@ -74,14 +74,19 @@ def parse_aws_release_calendar(page: str) -> list[CalItem]:
         raise RuntimeError("Failed to find minor version section")
 
     # the first table in the minor version section is the one we want
-    if not (version_table := minor_version_section.find_next("table")):
+    if not (version_table := minor_version_section.find_all_next("table", limit=2)):
```

Comment:
It would be better if we could configure the limit based on the RDS type, so that we can set it to 2 for MySQL only.

Reply:
So I tried that way, but it's throwing a uv error for `uv run mypy` that the function is getting too complex. The other way was to break it up into different functions which is a lot of change.